lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php

```diff
@@ -25,6 +25,8 @@
 use Magento\Framework\Mail\TransportInterfaceFactory;
 use Magento\Framework\ObjectManagerInterface;
 use Magento\Framework\Phrase;
+use Zend\Mime\Mime;
+use Zend\Mime\PartFactory;
 
```

Comment:
If  I am not wrong, on Magento v2.4-develop the reference to `Zend` namespace should be changed to `Laminas`.

Reply:
Thanks fo pointing that, I will change that in the next commit and prepare tests.

Reply:
Changed `Zend\Mime\Mime` to `Laminas\Mime\Mime`. Removed `Zend\Mime\PartFactory` from constructor, since generated Factory class is not valid, I've used standard object initialization instead - there is no real need for Factory here.